apps/telegram-bot/core/api_client.py

```
@@ -1,17 +1,17 @@
 import httpx
-from core.config import API_SERVER_URL, API_KEY, API_KEY_NAME
+from core.config import settings
 from fastfetchbot_shared.utils.logger import logger
 
 
 async def get_item(url: str, ban_list: list = None, **kwargs) -> dict:
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)